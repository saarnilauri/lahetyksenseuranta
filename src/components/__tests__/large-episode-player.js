import React from "react";
import renderer from "react-test-renderer";

import LargeEpisodePlayer from "../large-episode-player";

const episodeData = {
  title: "Seurantakoodi Mk 16:15-16",
  episodeNumber: 1,
  slug: "seurantakoodi-mk-16-15-16",
  published: "25.11.2020",
  excerpt: {
    excerpt:
      "Podcastin avausjakso. Kuuntele mistä podcastissä on kyse ja ota käyttöön ensimmäinen lähetyksenseurantakoodi.",
  },
  podbeanUrl:
    "https://www.podbean.com/media/player/i7h2y-f050a8?from=pb6admin&download=1&version=1&auto=0&share=0&download=1&rtl=0&fonts=Helvetica&skin=2&pfauth=&btn-skin=109",
  tags: [{ title: "Lähetyskäsky" }],
  bibleReference: [
    {
      title: "Markuksen evankeliumi 16:15-16",
      shortName: "Mk 16:15-16",
      text: { text: "Test verse" },
    },
  ],
  image: {
    fluid: {
      tracedSVG:
        "data:image/svg+xml,%3csvg%20xmlns='http://www.w3.org/2000/svg'%20width='350'%20height='196'%20viewBox='0%200%20350%20196'%20preserveAspectRatio='none'%3e%3cpath%20d='M0%2098v98h351V0H0v98m289-87c-2%203%200%205%203%203h9a2992%202992%200%200112-2c-1-4-2-5-5-3h-1c-2-2-5-1-5%202l-1%202-2-2c-3-4-7-4-10%200m27-2l-2%203c1%203%204%203%208%202h5c5%202%2011-2%206-5-3-1-8%200-9%202h-1V9c0-1-4-2-7%200m-148%202l-7%201-8%202c-2%201-17%203-34%203l-10%202-2%201-3%203-4%203-2%202c-2%202-3%203-5%203l-5%202-10%206c-6%202-6%203-7%205l1%203c1%202-3%205-6%205s-6%201-7%203v3c0%203-5%204-22%204-19%201-20%201-18%207%202%204%204%2016%204%2021-1%204%200%206%206%209l3%203%203%205%202%207%201%202h1l2%202c2%200%203%201%203%202l2%201%201-1c-1-1%200-1%201-1v1l4%201h4l9%201%2014%201%2018%206%201%201%202%201c2%201%203%202%204%201s4%200%204%201l2%201v3c-2%201-2%202%200%207l2%205c-1%200-4%207-3%208l-2%202-1%202%201%202%201%201%201-1%202%201%201%202h4c2%200%203%200%202%201s-1%201%201%201c2%201%202%200%202-1-1-1%201-2%202%200l2%201v1h1l2%201c-1%201%206%202%209%200l6-1c6%200%209-2%205-3-1-1-1-1%201-1l4%201v1l-3%201%201%201c1-1%203%200%203%201l5%201c3%200%2010%203%207%203l-1%201h4c1%202%200%202-2%202h-3l2%201c4%201%208-1%208-4%200-2%201-3%2010-3h25l14%204h1l1%201v-3l-1-2%203-2h3l3%201-1-2v-1c1%200%202%200%201-1l1-2%202-2c0-1%204-4%205-2h2c2-1%202-1%203%201h1c1-2%207-2%208-1h3c2-1%202-1%202%201%200%201%206%201%209-1h4l4-1%202-2v4l3-1c1-1%203-2%2010-2%202%200%202%200%201-1-2-2%201-3%205-2l9%201c6%200%206%200%201%202s-9%205-6%205l2-1h1l4-1c1-1%203-3%205-3l3-2h5c2-2%201-3-1-3l-2-3h1c0%202%200%202%202%200v-2l-2%201c0-2%209-9%2010-7s5%202%206%200l2-1c1%201%202%200%203-1h2l2-5c0-5%201-6%202-6%201-1%202-1%201-2h-4c-4-1-4-1-3%201l-1%201h-4c-1%201-1%201-1-1%201-2%200-3-2%200l-2%201%201-2v-2l-3%201h-1v-6h-1c-1%201-1%200-1-2l-1-3-1-1h1l1-2v-4l1-1c0%202%204%204%205%202v-3l-1-1h-2v-1l1-1c0-1-2%200-3%202l-3%202V97c1-1%202-2%203-1h3l-2-1c-2%200-2%200%200-4l3-4%201-1v-4%201c-2-1-6%205-6%207v1l-1%202h-1v-1l-1-1-1%201h-1l2-3%202-1h-2c-2%200-3-1%200-2%202-1%201-4-1-5-4%200-4-2%200-1%201%200%202%200%201-1-2%200-2%200-1-1v-2c1-2%201-2-1-2s-5-3-5-4c1-2-3-4-8-4-4%200-4%200-2-1v-2c0-1-2-4-4-4l-3-6c0-2-2-3-4-3l-3-2c-1-1-2-2-3-1l-1-1-1-2-6-3c-2-2-5-4-6-3v-2h-1l-2-1-5-4-4-2h-1l-5-1-5-1-2-1-5-1-6-1-7-2a171%20171%200%2001-18-4l-5-2-8-2-5-1-10-1-10-1h-8m-3%2050c-3%201-4%203-2%205%201%201%201%201-1%201s-2%200-1%201v1l3%201c4%201%204%203%201%204l-4%201-3%201h3c2-1%203%200%203%201v3c-1-1-1-1-2%201l-1%201c0-2-1-1-2%202l-3%208c-2%204-3%206-1%206l-1%203-3%204-1%201v4l-1%204c-2%200-1%204%201%205v1l-1%203v7c-1%202%201%206%204%207l4%203%201%201c1-1%201%200%201%201%200%202%205%207%206%206l1%201-3%201c-3%200-2%201%202%202l3%201%201%202%201%203%201%203%201%201v1l1-2h7c5%200%205-1%205-3l1-2%201-1-1-2-1-4%201-4h1l2%202v-1l-1-2-2-3c-1-1-1-1-1%201%201%203%200%202-3-4l-2-5c1%200%201-1-1-1-3-2-4-14%200-18l2-4c3-18%204-21%206-21l2-2%202-2c1-1%201-1-2-2-2-1-3-3-2-3l-5-1c-2%200-1-2%202-3h2l-3-2-3-3-3-2-2-3c1-2%200-4-3-5h-7m133%206l-1%202-1%204c-1%201%200%205%202%204l1%203c0%205%201%208%203%207h3c3%201%203%201%204-1l1-2c1-1%201-1-1-2l-2-1-2-1c-1-1%200-1%201-1l3%201%201%201%201-2c0-2-1-2-2-2l-2-2h2c1%200%202%200%202-2l-4-1c-7%201-11-1-7-4v-2l-2%201m-85%206v2c-1%202-2%202-2%200s0-2-1-1c-2%202-2%205%200%205v2c0%202%201%205%203%205l1-1c0-2%203-2%204-1%202%204%208%206%208%204-1-2-2-4-2-2l-4-3c-3-3-4-4-3-6l-1-3c-1-1-1%200-1%201l-1-1-1-1M46%20132h-3v31h19l1-2c0-2%200-2-6-2h-6v-13c0-15%200-15-5-14m22%202c-4%203-3%207%206%2015l7%208c-1%203-6%202-7-1l-1-2h-4c-8%200-4%208%204%209%2015%203%2021-7%2010-15l-7-6c-3-3-3-3-2-5h4l1%201h2c0%201%201%202%204%202%204%200%205-2%202-5-2-3-15-4-19-1m139%2017l-3%201h-2l-1%203h-6c-3-1-5%200-5%201%200%202%205%203%209%202h3c-2%201-1%201%203%201a300%20300%200%20017-2c-1-1%200-1%203-1s4%200%203-2l-4-1c-5%200-5%200-2%201%201%201%201%201-2%201s-4-1-2-3l1-2-2%201'%20fill='%23d3d3d3'%20fill-rule='evenodd'/%3e%3c/svg%3e",
      aspectRatio: 1.7857142857142858,
      src:
        "//images.ctfassets.net/1ytenrlcq9nc/1fbBUWeiHM0e7HWZcPRta2/19318bedb93604a4678f880aea025606/jakso-1.jpg?w=350&h=196&q=50&fit=scale",
      srcSet:
        "//images.ctfassets.net/1ytenrlcq9nc/1fbBUWeiHM0e7HWZcPRta2/19318bedb93604a4678f880aea025606/jakso-1.jpg?w=88&h=49&q=50&fit=scale 88w,\n//images.ctfassets.net/1ytenrlcq9nc/1fbBUWeiHM0e7HWZcPRta2/19318bedb93604a4678f880aea025606/jakso-1.jpg?w=175&h=98&q=50&fit=scale 175w,\n//images.ctfassets.net/1ytenrlcq9nc/1fbBUWeiHM0e7HWZcPRta2/19318bedb93604a4678f880aea025606/jakso-1.jpg?w=350&h=196&q=50&fit=scale 350w,\n//images.ctfassets.net/1ytenrlcq9nc/1fbBUWeiHM0e7HWZcPRta2/19318bedb93604a4678f880aea025606/jakso-1.jpg?w=525&h=294&q=50&fit=scale 525w,\n//images.ctfassets.net/1ytenrlcq9nc/1fbBUWeiHM0e7HWZcPRta2/19318bedb93604a4678f880aea025606/jakso-1.jpg?w=700&h=392&q=50&fit=scale 700w,\n//images.ctfassets.net/1ytenrlcq9nc/1fbBUWeiHM0e7HWZcPRta2/19318bedb93604a4678f880aea025606/jakso-1.jpg?w=1050&h=588&q=50&fit=scale 1050w,\n//images.ctfassets.net/1ytenrlcq9nc/1fbBUWeiHM0e7HWZcPRta2/19318bedb93604a4678f880aea025606/jakso-1.jpg?w=1400&h=784&q=50&fit=scale 1400w",
      sizes: "(max-width: 350px) 100vw, 350px",
    },
  },
  description: {
    raw:
      '{"data":{},"content":[{"data":{},"content":[{"data":{},"marks":[],"value":"Podcastin avausjakso. Kuuntele mistä podcastissä on kyse ja ota käyttöön ensimmäinen lähetyksenseurantakoodi.","nodeType":"text"}],"nodeType":"paragraph"},{"data":{},"content":[{"data":{},"marks":[],"value":"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque erat ligula, sollicitudin sed nibh vel, dictum elementum nisl. Aliquam sollicitudin odio sed augue ultrices, eu semper lacus porttitor. Pellentesque pulvinar lacus dui, sit amet rhoncus justo pretium at. Vivamus quis lectus sollicitudin, tempor est a, egestas dui. Sed vitae mi elit. Etiam et dictum diam. Nunc gravida fringilla odio accumsan accumsan. Nunc in sapien tincidunt metus congue laoreet. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Maecenas pretium, dolor non fermentum hendrerit, felis lorem mattis nisi, a consectetur tortor risus in turpis. Suspendisse sollicitudin sed mauris nec tincidunt. Maecenas rhoncus neque erat, a tempus purus tempus eget. In nec tincidunt odio, ut sagittis nibh.","nodeType":"text"}],"nodeType":"paragraph"},{"data":{},"content":[{"data":{},"marks":[],"value":"Proin id ex blandit, fringilla mi in, suscipit dolor. Curabitur maximus neque augue, ut interdum tellus vestibulum vel. Pellentesque bibendum mi et quam efficitur, sit amet semper arcu efficitur. Curabitur nulla velit, volutpat volutpat neque sed, elementum aliquet leo. Ut imperdiet nunc arcu, sed feugiat nisl cursus ut. Curabitur dignissim commodo erat. Mauris egestas felis sit amet enim faucibus, id molestie sem elementum. Quisque rhoncus a justo sit amet egestas. Nam vel libero dictum, tempor ante nec, venenatis dui. Proin eget tempor lorem. Proin tincidunt varius tellus in posuere. Phasellus vestibulum eleifend lorem a pulvinar. Integer tristique nulla tempus augue fermentum, quis sollicitudin arcu maximus. Etiam pretium, lectus sed euismod congue, quam ante ornare felis, sit amet placerat risus nibh et sem. Nam ullamcorper libero sed felis dignissim, non vulputate sem euismod. Morbi condimentum nunc quis felis pulvinar, nec pellentesque mauris molestie.","nodeType":"text"}],"nodeType":"paragraph"}],"nodeType":"document"}',
  },
};

describe("LargeEpisodePlayer", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<LargeEpisodePlayer episode={episodeData} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
