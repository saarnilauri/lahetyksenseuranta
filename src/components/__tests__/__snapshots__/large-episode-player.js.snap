// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LargeEpisodePlayer renders correctly 1`] = `
<div
  className="container mx-auto"
>
  <div
    className="bg-gray-700 py-10 px-10 text-white"
  >
    <div
      class="flex flex-row content-center items-center mb-10"
    >
      <div
        className="episode-number text-4xl py-4 px-4 bg-white text-gray-900 uppercase mr-10"
      >
        Jakso 
        1
      </div>
      <div>
        <h2
          className="text-6xl mb-5 font-bold"
        >
          Seurantakoodi Mk 16:15-16
        </h2>
        <div
          class="flex flex-row pl-100"
        >
          <div
            className="mr-2"
          >
            <svg
              height="30"
              viewBox="0 0 28 30"
              width="28"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m2.509 12.432h22.944v14.85h-22.944z"
                fill="#fff"
              />
              <path
                d="m3.334 3.925h21.042c.924 0 1.672.508 1.672 1.134v6.239c0 .626-.748 1.134-1.672 1.134h-21.042c-.924 0-1.673-.508-1.673-1.134v-6.239c0-.626.749-1.134 1.673-1.134z"
                fill="#f71025"
              />
              <g
                fill="#0e0c0c"
              >
                <path
                  d="m22.868 2.804c2.572 0 4.632 2.084 4.632 4.626v17.944c0 2.542-2.06 4.626-4.632 4.626h-17.736c-2.572 0-4.632-2.084-4.632-4.626v-17.944c0-2.542 2.06-4.626 4.632-4.626zm0 2.523h-17.736c-1.101 0-2.019.929-2.019 2.103v17.944c0 1.174.918 2.103 2.019 2.103h17.736c1.101 0 2.019-.929 2.019-2.103v-17.944c0-1.174-.918-2.103-2.019-2.103z"
                />
                <path
                  d="m9.645 0c.722 0 1.307.565 1.307 1.262v6.729c0 .696-.585 1.261-1.307 1.261-.721 0-1.306-.565-1.306-1.261v-6.729c0-.697.585-1.262 1.306-1.262z"
                />
                <path
                  d="m18.355 0c.721 0 1.306.565 1.306 1.262v6.729c0 .696-.585 1.261-1.306 1.261-.722 0-1.307-.565-1.307-1.261v-6.729c0-.697.585-1.262 1.307-1.262z"
                />
              </g>
              <path
                d="m5.267 14.477h2.535v2.448h-2.535z"
              />
              <path
                d="m10.147 14.477h2.536v2.448h-2.536z"
              />
              <path
                d="m15.027 14.477h2.536v2.448h-2.536z"
              />
              <path
                d="m19.907 14.477h2.536v2.448h-2.536z"
              />
              <path
                d="m5.267 18.672h2.535v2.449h-2.535z"
              />
              <path
                d="m10.147 18.672h2.536v2.449h-2.536z"
              />
              <path
                d="m15.027 18.672h2.536v2.449h-2.536z"
              />
              <path
                d="m19.907 18.672h2.536v2.449h-2.536z"
              />
              <path
                d="m5.267 22.867h2.535v2.449h-2.535z"
              />
              <path
                d="m10.147 22.867h2.536v2.449h-2.536z"
              />
              <path
                d="m15.027 22.867h2.536v2.449h-2.536z"
              />
              <path
                d="m19.907 22.867h2.536v2.449h-2.536z"
              />
            </svg>
          </div>
          <div
            className="mr-5"
          >
            <p>
              25.11.2020
            </p>
          </div>
          <div
            className="mr-2"
          >
            <svg
              height="30"
              viewBox="0 0 30 30"
              width="30"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m15 29c-7.732 0-14-6.268-14-14s6.268-14 14-14 14 6.268 14 14-6.268 14-14 14z"
                fill="#fff"
              />
              <path
                d="m15 0c8.284 0 15 6.716 15 15s-6.716 15-15 15-15-6.716-15-15 6.716-15 15-15zm0 2.857c-6.706 0-12.143 5.437-12.143 12.143s5.437 12.143 12.143 12.143 12.143-5.437 12.143-12.143-5.437-12.143-12.143-12.143z"
              />
              <path
                d="m15 4.996c.526 0 .952.426.952.952v9.052c0 .526-.426.952-.952.952s-.952-.426-.952-.952v-9.052c0-.526.426-.952.952-.952z"
              />
              <path
                d="m19.294 14.048c.526 0 .952.426.952.952s-.426.952-.952.952h-4.297c-.526 0-.952-.426-.952-.952s.426-.952.952-.952z"
              />
            </svg>
          </div>
          <div
            className="mr-5"
          >
            <p>
              19m 20s
            </p>
          </div>
          <div
            className="mr-2"
          >
            <svg
              height="30"
              viewBox="0 0 40 30"
              width="40"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m.714 4.415h2.674s.711-2.294.711-2.294 4.444-1.587 6.576-1.235c2.133.353 9.42 4.058 9.42 4.058s5.51-4.058 9.597-4.058h4.088s2.666 1.765 2.666 1.765v1.588h2.133s.178 24.877.178 24.877-38.043-.176-38.043-.176.897-24.525 0-24.525z"
                fill="#fff"
              />
              <path
                d="m30.078 0c1.413 0 2.972.2 4.537.53.844.19.536.112.929.213.03-.075.114.016.167.043l.418.107c.172.047.322.091.433.126.058.018.106.034.146.049s.04.015.147.068c.047.026.047.026.134.09.134.092.134.092.348.587-.006.464-.022.929-.028 1.394h1.731c.53 0 .96.429.96.959v25.094c0 .53-.43.96-.96.96h-38.08c-.53 0-.96-.43-.96-.96v-25.094c0-.53.43-.959.96-.959h1.731c.003-.465-.029-.93-.028-1.395.214-.494.214-.494.348-.586.087-.064.087-.064.134-.09.108-.053.108-.053.147-.068.041-.015.088-.031.146-.049.111-.035.261-.079.433-.126l.418-.107c.053-.027.137-.118.168-.043.392-.101.084-.023.928-.213 1.565-.33 3.124-.53 4.537-.53 2.752 0 4.132.374 6.232 1.425.455.227 1.043.563 1.739.987.101.061.203.123.307.188.507.313 1.034.647 1.549.982l.251.163.251-.163c.515-.335 1.042-.669 1.549-.982.104-.065.206-.127.307-.188.696-.424 1.284-.76 1.739-.987 2.1-1.051 3.48-1.425 6.232-1.425zm-27.382 5.126h-.777v23.175h16.286c-.512-.251-1.06-.518-1.551-.756-.093-.045-.093-.045-.185-.09-.665-.326-1.179-.585-1.6-.807-1.55-.821-3.626-1.314-5.456-1.314-.785 0-2.086.177-3.478.452-.567.113-1.103.233-1.493.333-.173.045-.306.083-.382.109-.011.004-.011.004.016-.01l.098-.06c-.567.278-.736.285-1.242-.087-.187-.184-.289-.422-.277-.683h-.004c0-.02 0-.02.001-.118 0-.136 0-.136.001-.342.002-.373.003-.8.005-1.274.006-1.355.011-2.818.016-4.335.006-1.951.011-3.848.015-5.636.007-3.292.009-6.034.005-8.045zm34.608 0 .002.512c-.004 2.011-.002 4.753.005 8.045.004 1.788.009 3.685.015 5.636.005 1.517.011 2.98.016 4.335.002.474.003.901.005 1.274.001.206.001.206.001.342.001.098.001.098.001.118h-.004c.012.261-.09.499-.277.683-.506.372-.675.365-1.242.087l.098.06c.027.014.027.014.016.01-.075-.026-.209-.064-.382-.109-.39-.1-.926-.22-1.493-.333-1.392-.275-2.692-.452-3.478-.452-1.83 0-3.906.493-5.456 1.314-.421.222-.935.481-1.6.807-.092.045-.092.045-.185.09-.491.239-1.039.505-1.551.756h16.286v-23.175zm-27.382-3.207c-1.262 0-2.696.184-4.141.489-.893.201-.499.104-1.185.28l.003.285c.007.621.012 1.515.014 2.661.005 2.014.003 4.759-.005 8.053-.003 1.789-.009 3.686-.015 5.638-.005 1.518-.01 2.981-.016 4.337l-.002.454c.691-.155.362-.085.987-.212 1.506-.298 2.91-.489 3.851-.489 2.136 0 4.521.566 6.354 1.536.4.212.898.463 1.547.781.091.044.091.044.182.089.411.199.906.44 1.34.652-.023-1.427-.015-.771-.025-1.968-.013-2.009-.007-4.745.014-8.027.012-1.782.028-3.671.047-5.615.015-1.512.031-2.97.046-4.319.011-.85.006-.449.015-1.203l-.229-.149c-.504-.328-1.018-.655-1.512-.959-.1-.062-.199-.123-.296-.182-.655-.398-1.203-.711-1.6-.91-1.85-.925-2.944-1.222-5.374-1.222zm20.156 0c-2.43 0-3.524.297-5.374 1.222-.397.199-.945.512-1.6.91-.097.059-.196.12-.296.182-.494.304-1.008.631-1.512.959l-.229.149c.009.754.004.353.015 1.203.016 1.349.031 2.807.046 4.319.019 1.944.035 3.833.047 5.615.021 3.282.027 6.018.014 8.027-.01 1.197-.002.541-.024 1.968.434-.212.928-.453 1.339-.652.091-.045.091-.045.182-.089.649-.318 1.148-.569 1.547-.781 1.833-.97 4.218-1.536 6.354-1.536.942 0 2.345.191 3.851.489.625.127.296.057.987.212l-.002-.454c-.006-1.356-.011-2.819-.016-4.337-.006-1.952-.012-3.849-.015-5.638-.008-3.294-.01-6.039-.005-8.053.002-1.146.007-2.04.014-2.661l.003-.285c-.686-.176-.292-.079-1.185-.28-1.445-.305-2.879-.489-4.141-.489zm-25.595 23.834-.249.355.067-.059c.146-.179.085-.081.182-.296zm31.034 0c.097.215.036.117.182.296l.067.059zm-30.958-.282h-.001l-.027.175c.039-.086.021-.031.028-.175zm30.882 0c.007.144-.011.089.028.175z"
              />
              <path
                d="m11.555 4.151c.472.138.718.46.766.94v2.737h2.553c.53 0 .959.43.959.96s-.429.959-.959.959h-2.553v9.947c0 .53-.43.96-.959.96-.53 0-.96-.43-.96-.959v-9.948h-2.553c-.53 0-.959-.429-.959-.959s.429-.96.959-.96h2.553v-2.737c0-.53.43-.96.96-.96z"
              />
            </svg>
          </div>
          <div
            className="mr-5"
          >
            <button
              aria-controls="popover-1"
              aria-expanded={false}
              aria-haspopup="dialog"
              className="py-1 px-2 hover:underline"
              data-radix-popover-trigger=""
              onClick={[Function]}
              type="button"
            >
              Mk 16:15-16
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      className="episode-excerpt"
    >
      Podcastin avausjakso. Kuuntele mistä podcastissä on kyse ja ota käyttöön ensimmäinen lähetyksenseurantakoodi.
    </div>
    <div
      className="app"
    >
      <section
        className="section"
      >
        <h1>
          ReactPlayer Demo
        </h1>
        <div
          className="player-wrapper"
        >
          <div
            className="react-player"
            style={
              Object {
                "height": "100%",
                "width": "100%",
              }
            }
          />
        </div>
        <table>
          <tbody>
            <tr>
              <th>
                Controls
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  Stop
                </button>
                <button
                  onClick={[Function]}
                >
                  Play
                </button>
                <button>
                  Fullscreen
                </button>
              </td>
            </tr>
            <tr>
              <th>
                Speed
              </th>
              <td>
                <button
                  onClick={[Function]}
                  value={1}
                >
                  1x
                </button>
                <button
                  onClick={[Function]}
                  value={1.5}
                >
                  1.5x
                </button>
                <button
                  onClick={[Function]}
                  value={2}
                >
                  2x
                </button>
              </td>
            </tr>
            <tr>
              <th>
                Seek
              </th>
              <td>
                <input
                  max={0.999999}
                  min={0}
                  onChange={[Function]}
                  onMouseDown={[Function]}
                  onMouseUp={[Function]}
                  step="any"
                  type="range"
                  value={0}
                />
              </td>
            </tr>
            <tr>
              <th>
                Volume
              </th>
              <td>
                <input
                  max={1}
                  min={0}
                  onChange={[Function]}
                  step="any"
                  type="range"
                  value={0.8}
                />
              </td>
            </tr>
            <tr>
              <th>
                <label
                  htmlFor="controls"
                >
                  Controls
                </label>
              </th>
              <td>
                <input
                  checked={false}
                  id="controls"
                  type="checkbox"
                />
                <em>
                    Requires player reload
                </em>
              </td>
            </tr>
            <tr>
              <th>
                <label
                  htmlFor="muted"
                >
                  Muted
                </label>
              </th>
              <td>
                <input
                  checked={false}
                  id="muted"
                  onChange={[Function]}
                  type="checkbox"
                />
              </td>
            </tr>
            <tr>
              <th>
                <label
                  htmlFor="loop"
                >
                  Loop
                </label>
              </th>
              <td>
                <input
                  checked={false}
                  id="loop"
                  type="checkbox"
                />
              </td>
            </tr>
            <tr>
              <th>
                <label
                  htmlFor="light"
                >
                  Light mode
                </label>
              </th>
              <td>
                <input
                  checked={false}
                  id="light"
                  type="checkbox"
                />
              </td>
            </tr>
            <tr>
              <th>
                Played
              </th>
              <td>
                <progress
                  max={1}
                  value={0}
                />
              </td>
            </tr>
            <tr>
              <th>
                Loaded
              </th>
              <td>
                <progress
                  max={1}
                  value={0}
                />
              </td>
            </tr>
          </tbody>
        </table>
      </section>
      <section
        className="section"
      >
        <table>
          <tbody>
            <tr>
              <th>
                YouTube
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  Test A
                </button>
                <button
                  onClick={[Function]}
                >
                  Test B
                </button>
                <button
                  onClick={[Function]}
                >
                  Playlist
                </button>
              </td>
            </tr>
            <tr>
              <th>
                SoundCloud
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  Test A
                </button>
                <button
                  onClick={[Function]}
                >
                  Test B
                </button>
                <button
                  onClick={[Function]}
                >
                  Playlist
                </button>
              </td>
            </tr>
            <tr>
              <th>
                Facebook
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  Test A
                </button>
                <button
                  onClick={[Function]}
                >
                  Test B
                </button>
              </td>
            </tr>
            <tr>
              <th>
                Vimeo
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  Test A
                </button>
                <button
                  onClick={[Function]}
                >
                  Test B
                </button>
              </td>
            </tr>
            <tr>
              <th>
                Twitch
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  Test A
                </button>
                <button
                  onClick={[Function]}
                >
                  Test B
                </button>
                <button
                  onClick={[Function]}
                >
                  Test C
                </button>
              </td>
            </tr>
            <tr>
              <th>
                Streamable
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  Test A
                </button>
                <button
                  onClick={[Function]}
                >
                  Test B
                </button>
              </td>
            </tr>
            <tr>
              <th>
                Wistia
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  Test A
                </button>
                <button
                  onClick={[Function]}
                >
                  Test B
                </button>
                <button
                  onClick={[Function]}
                >
                  Test C
                </button>
              </td>
            </tr>
            <tr>
              <th>
                DailyMotion
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  Test A
                </button>
                <button
                  onClick={[Function]}
                >
                  Test B
                </button>
              </td>
            </tr>
            <tr>
              <th>
                Mixcloud
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  Test A
                </button>
                <button
                  onClick={[Function]}
                >
                  Test B
                </button>
              </td>
            </tr>
            <tr>
              <th>
                Vidyard
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  Test A
                </button>
                <button
                  onClick={[Function]}
                >
                  Test B
                </button>
              </td>
            </tr>
            <tr>
              <th>
                Kaltura
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  Test A
                </button>
                <button
                  onClick={[Function]}
                >
                  Test B
                </button>
              </td>
            </tr>
            <tr>
              <th>
                Files
              </th>
              <td>
                <button
                  onClick={[Function]}
                >
                  mp4
                </button>
                <button
                  onClick={[Function]}
                >
                  webm
                </button>
                <button
                  onClick={[Function]}
                >
                  ogv
                </button>
                <button
                  onClick={[Function]}
                >
                  mp3
                </button>
                <br />
                <button
                  onClick={[Function]}
                >
                  HLS (m3u8)
                </button>
                <button
                  onClick={[Function]}
                >
                  DASH (mpd)
                </button>
              </td>
            </tr>
            <tr>
              <th>
                Custom URL
              </th>
              <td>
                <input
                  placeholder="Enter URL"
                  type="text"
                />
                <button
                  onClick={[Function]}
                >
                  Load
                </button>
              </td>
            </tr>
          </tbody>
        </table>
        <h2>
          State
        </h2>
        <table>
          <tbody>
            <tr>
              <th>
                url
              </th>
              <td
                className="faded"
              >
                null
              </td>
            </tr>
            <tr>
              <th>
                playing
              </th>
              <td>
                false
              </td>
            </tr>
            <tr>
              <th>
                volume
              </th>
              <td>
                0.800
              </td>
            </tr>
            <tr>
              <th>
                played
              </th>
              <td>
                0.000
              </td>
            </tr>
            <tr>
              <th>
                loaded
              </th>
              <td>
                0.000
              </td>
            </tr>
            <tr>
              <th>
                duration
              </th>
              <td>
                <time
                  dateTime="P0S"
                >
                  0:00
                </time>
              </td>
            </tr>
            <tr>
              <th>
                elapsed
              </th>
              <td>
                <time
                  dateTime="P0S"
                >
                  0:00
                </time>
              </td>
            </tr>
            <tr>
              <th>
                remaining
              </th>
              <td>
                <time
                  dateTime="P0S"
                >
                  0:00
                </time>
              </td>
            </tr>
          </tbody>
        </table>
      </section>
    </div>
  </div>
</div>
`;
